Gemfile

```diff
@@ -7,3 +7,4 @@ gemspec
 
 gem 'pry'
 gem 'pry-byebug'
+gem 'hocon', '~> 1.2', require: false
```

Comment:
What does ``require: false`` do?

Reply:
When you call Bundler.setup it will attempt to autorequire all your gems. We like to manually require them to ensure our classes that depend on something do it explicitly. We don't do it above so when debugging (we only do Bundle.setup in dev environments) we can easily drop pry statements in there.

See https://bundler.io/man/gemfile.5.html#REQUIRE-AS and our usage in
https://github.com/puppetlabs/puppetserver-ca-cli/blob/master/exe/puppetserver-ca#L6

Reply:
thanks!